app/components/edit-claim-conditions/EditClaimConditions.tsx

```
@@ -0,0 +1,300 @@
+      //   metadata: {
+      //     name: formData.phaseName, // Name of the sale's phase
+      //   },
+      // });
+
+      return true;
+    } catch (err) {
+      console.error({ err });
+
+      toast({
+        title: 'Set Claim Conditions',
+        description: `Failed to set claim conditions`,
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)